xmake/actions/package/local/main.lua

```diff
@@ -142,6 +143,32 @@ function _package_library(target)
         end
     end
 
+    -- copy modules
+    if target:data("cxx.has_modules") then
+        import(".rules.c++.modules.modules_support.compiler_support")
+        import(".rules.c++.modules.modules_support.builder")
```

Comment:
please move it to header, and use `import("rules.c++.modules.modules_support.builder", {rootdir = os.programdir()})`

Reply:
and use `module_builder` as alias